src/data/selector.ts

```diff
@@ -352,6 +352,7 @@ export interface NumberSelector {
 interface ObjectSelectorField {
   selector: Selector;
   label?: string;
+  helper?: string;
```

Comment:
What's `helper` for?

Reply:
The description is called helper in front-end. Hum... We should also call it description in the selector as it can be used in core. I'll update that.